x/coinswap/client/cli/query.go

```diff
@@ -0,0 +1,93 @@
+package cli
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/spf13/cobra"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/context"
+	"github.com/cosmos/cosmos-sdk/codec"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/version"
+	"github.com/cosmos/cosmos-sdk/x/coinswap/internal/types"
+)
+
+// GetQueryCmd returns the cli query commands for this module
+func GetQueryCmd(queryRoute string, cdc *codec.Codec) *cobra.Command {
+	coinswapQueryCmd := &cobra.Command{
+		Use:                        types.ModuleName,
+		Short:                      "Querying commands for the coinswap module",
+		DisableFlagParsing:         true,
+		SuggestionsMinimumDistance: 2,
+		RunE:                       client.ValidateCmd,
+	}
+	coinswapQueryCmd.AddCommand(client.GetCommands(
+		GetCmdQueryLiquidity(queryRoute, cdc),
+		GetCmdQueryParams(queryRoute, cdc))...)
+
+	return coinswapQueryCmd
+}
+
+// GetCmdQueryLiquidity implements the liquidity query command
+func GetCmdQueryLiquidity(queryRoute string, cdc *codec.Codec) *cobra.Command {
```

Comment:
This command will need to take in a denomination to query liquidity for as an argument

Reply:
Yes, will add it.